### ISSUE TITLE
eLoS3 Formula does not include FlyBoats

### ISSUE THREAD

**Issue by bamboo3250** [KC3Kai/KC3Kai#687]
Wait, I need to confirm again,.

**Comment by ethrundr** [comment]
Ah I noticed this several times before while leveling Akistushima, but forget to submit an issue.
Yes, this is happening.

**Comment by bamboo3250** [comment]
Yeah, I saw it updated twice in a row. Don't know if it's intended or not since I'm not familiar with LoS formula.

**Comment by bamboo3250** [comment]
btw, it only happens with LoS new, the Sum and Old don't seem to have this issue.

**Comment by ethrundr** [comment]
I believe we just forget to add it, or somehow on the new formula, Taitei-chan is not included.

**Comment by dragonjet** [comment]
ah well, the formula doesn't have something for flyboats

```
= Dive Bomber LoS x (1.04)
+ Torpedo Bomber LoS x (1.37)
+ Carrier-based Recon Plane LoS x (1.66)
+ Recon Seaplane LoS x (2.00)
+ Seaplane Bomber LoS x (1.78)
+ Small Radar LoS x (1.00)
+ Large Radar LoS x (0.99)
+ Searchlight LoS x (0.91)
+ √(base LoS of each ship) x (1.69)
+ (HQ Lv. rounded up to the next multiple of 5) x (-0.61)
```

since the formula was made long ago.
If we will add flyboats, we don't know the multiplier XD

**Comment by bamboo3250** [comment]
Are those multipliers extracted by some kind of statistics analysis method? I suck at Statistics xD

**Comment by dragonjet** [comment]
I just got it from the wikia :v
and the wikia just got it from JP wiki
and the JP wiki got it from aliens idk :laughing:

**Comment by nicky008** [comment]
and the aliens got it from the RNG-goddess, right?

**Comment by dragonjet** [comment]
lol and the RNG-goddess got it from CDRW

**Comment by ethrundr** [comment]
is there any way to extract the final eLoS stat from the api_data?
if there any, we can calculate the multiplier by ourselves, actually.....

**Comment by dragonjet** [comment]
ah nope, compass data does not show eLoS, all of these are derived from thousands of JP tests I guess
